packages/framework/src/Form/Admin/Order/OrderFormType.php

```diff
@@ -612,20 +639,30 @@ private function createPaymentTransactionsGroup(FormBuilderInterface $builder, O
 
     /**
      * @param \Symfony\Component\Form\FormBuilderInterface $builder
-     * @param \Shopsys\FrameworkBundle\Model\Order\Withdrawal\WithdrawalRequest $withdrawalRequest
+     * @param \Shopsys\FrameworkBundle\Model\Order\Withdrawal\WithdrawalRequest|null $withdrawalRequest
      * @return \Symfony\Component\Form\FormBuilderInterface
      */
     private function createWithdrawalRequestGroup(
         FormBuilderInterface $builder,
-        WithdrawalRequest $withdrawalRequest,
+        ?WithdrawalRequest $withdrawalRequest,
     ): FormBuilderInterface {
+        $rowAttr = [
+            'class' => 'js-order-withdrawal-group',
+            'data-withdrawal-request-exists' => $withdrawalRequest !== null,
```

Comment:
Do we need both? I think we can safely use just the data attribute as a selector, or?

Reply:
fixed in c23f7b8776